admin/app/controllers/authenticated/certifications/certification/details.js

```diff
@@ -48,7 +48,7 @@ export default class CertificationDetailsController extends Controller {
   onStoreMarks() {
     this._markStore.storeState({
       score: (this.juryScore === false) ? this.score : this.juryScore,
-      marks: this.get('details.competences').reduce((marks, competence) => {
+      marks: this.details.competences.reduce((marks, competence) => {
```

Comment:
Quand le niveau de profondeur est grand + combiné à une méthode comme ici, on peut avoir des erreurs silencieuse. Tu peux t'en protéger comme ça 

```suggestion
      marks: this.details.get('competences').reduce((marks, competence) => {
```

Reply:
Vu ensemble : `this.details.competences` est initialement un tableau vide, qui sera ensuite rempli (ou pas) de competences. 
Done